L1Trigger/Phase2L1ParticleFlow/test/l1pfJetMetTreeProducer.py

```diff
@@ -1,31 +1,192 @@
+############################################################
```

Comment:
we're not supporting this cfg anymore for L1PF JetMET studies, so if you still want it to create pattern files for GTT it would be safer to rename it to avoid confusion

Reply:
@gpetruc, should I change it to be `patternFileProducer.py`? That's more or less all it's used for, I think.

Reply:
ok